pull/github.go

```diff
@@ -548,7 +548,12 @@ func (ghc *GitHubContext) loadPagedData() error {
 		}
 
 		for _, r := range q.Repository.PullRequest.Reviews.Nodes {
-			reviews = append(reviews, r.ToReview())
+			switch r.State {
+			case "COMMENTED":
```

Comment:
Maybe this should be `case string(githubv4.PullRequestReviewStateCommented):` instead?

I did not find any instance of using the https://github.com/shurcooL/githubv4 module constants anywhere in the code, and thus decided to use these strings instead, but happy to change that if you'd like.